src/plots/ternary/ternary.js

```diff
@@ -257,6 +285,9 @@ proto.adjustLayout = function(ternaryLayout, graphSize) {
     _this.clipDef.select('path').attr('d', triangleClip);
     _this.layers.plotbg.select('path').attr('d', triangleClip);
 
+    var triangleClipRelative = 'M0,' + h + 'h' + w + 'l-' + (w / 2) + ',-' + h + 'Z';
+    _this.clipDefRelative.select('path').attr('d', triangleClipRelative);
```

Comment:
heh, yet another consequence of when we had nested `<svg>` elements instead of clip paths - I suspect we could simplify a whole lot of stuff by not translating containers, but that's a big project. Until then we need stuff like this 👍